ext/json/tests/json_validate_005.phpt

```diff
@@ -0,0 +1,72 @@
+--TEST--
+json_validate() - compare against json_decode() for different types of inputs 
+--FILE--
+<?php
+
+$inputs = [
+    '""',
+    '"string"',
+    '1234',
+    '123.45',
+    '-123',
+    'null',
+    'true',
+    'false',
+    '{}',
+    '[]',
+    '-',
+    null,
+    '',
+];
+
+foreach ($inputs as $input) {
+    var_dump($input, json_decode($input), json_validate($input));
+}
+
+?>
+--EXPECTF--
+string(2) """"
+string(0) ""
+bool(true)
+string(8) ""string""
+string(6) "string"
+bool(true)
+string(4) "1234"
+int(1234)
+bool(true)
+string(6) "123.45"
+float(123.45)
+bool(true)
+string(4) "-123"
+int(-123)
+bool(true)
+string(4) "null"
+NULL
+bool(true)
+string(4) "true"
+bool(true)
+bool(true)
+string(5) "false"
+bool(false)
+bool(true)
+string(2) "{}"
+object(stdClass)#1 (0) {
+}
+bool(true)
+string(2) "[]"
+array(0) {
+}
+bool(true)
+string(1) "-"
+NULL
+bool(false)
+
+Deprecated: json_decode(): Passing null to parameter #1 ($json) of type string is deprecated in %s on line %d
+
+Deprecated: json_validate(): Passing null to parameter #1 ($json) of type string is deprecated in %s on line %d
```

Comment:
nit: This is just a consequence of https://wiki.php.net/rfc/deprecate_null_to_scalar_internal_arg in php 8.1 - we know why it happens and we're sure the parameter type of json_decode and json_validate won't change, so there's no reason to test it

I think you can just take out the tests of `null` and the deprecation - it's a deprecation warning common to all internal functions, it has nothing specific about the json module.

Reply:
Ok @TysonAndre Thanks for the information, I will take out the test of `null` and the deprecation. I will apply the changes once I get back home.

I ping you when ready.

Have a good day.

Reply:
@TysonAndre change done, had to remove the test case in 2 test files.

:heavy_check_mark: